src/integration-test/java/apptest/CreateProcessIT.java

```diff
@@ -30,6 +30,7 @@
 @WireMockAppTestSuite(files = "classpath:/CreateProcess/", classes = Application.class)
 class CreateProcessIT extends AbstractCamundaAppTest {
 	private static final int DEFAULT_TESTCASE_TIMEOUT_IN_SECONDS = 30;
+	private static final String MUNICIPALITY_ID = "/2281";
```

Comment:
If this variable should be named MUNICIPALITY_ID then it should not contain anything else than the municipality-ID, i.m.h.o.